### ISSUE TITLE
Midterm Peer Report (rz98)

### ISSUE THREAD

**Issue by ZRich098** [GraceJiangJinyi/ORIE4741_Project#11]
The group plans to model the impact of Covid-19 on the stock index in the US and China.  They plan to build supervised learning models to see the relationship between factors like number of Covid cases and deaths to predict the effect of industry indications such as stocks, consumer products, and healthcare.  They plan on using many model types like Linear Regression, SVR, and Neural Networks to see which can most accurately predict stock performance given Covid conditions.

Three things I liked:
1. There is definitely a lot of data available (daily measurements for cases and stocks for months), which will help model accuracy and avoiding overfitting. 
2. Accounting for possible time delays between Covid news and corresponding stock performances.  I never though about it, but news does take time to propagate, and I feel news is more likely to affect stocks.
3. The well defined schedule at the end of the project to keep the group on track.  

Three things I didn't like:
1. There will definitely be a lot of noise in the data, since stock prices are affected by a multitude of factors such as politics, etc. It will be hard to produce an accurate model.
2. I like the detailed plans for which models will be used, but I would have liked to see whether any initial modeling produced any results or thoughts on what the expected outputs of the model will look like. 
3. I feel like the group could have gone more in depth on what the features they chose looks like and why they chose those features.
